extension/extension/writing.js

```diff
@@ -138,6 +140,14 @@ function is_string(myVar) {
     }
 }
 
+function injectScript(file_path, tag) {
```

Comment:
Same comment here about the documentation for the function.

Reply:
I've added this